packages/Confirmation/src/components/TriggerButton/TriggerButton.js

```diff
@@ -1,40 +1,54 @@
 import React from "react";
 import PropTypes from "prop-types";
 import Button from "@paprika/button";
+import RawButton from "@paprika/raw-button";
```

Comment:
I will be following this PR up with a new proposal where we consolidate this `triggerButton` component with the trigger in the Dropdown component. At the moment I am just ensuring that they both use the same api for button type instead of using an `icon` prop here. Confirmation components `triggerButton` will now also support raw content as it's child.